CHANGEDB.php

```diff
@@ -923,4 +923,6 @@
 INSERT INTO `gibbonAction` (`gibbonModuleID`, `name`, `precedence`, `category`, `description`, `URLList`, `entryURL`, `defaultPermissionAdmin`, `defaultPermissionTeacher`, `defaultPermissionStudent`, `defaultPermissionParent`, `defaultPermissionSupport`, `categoryPermissionStaff`, `categoryPermissionStudent`, `categoryPermissionParent`, `categoryPermissionOther`) VALUES ((SELECT gibbonModuleID FROM gibbonModule WHERE name='Attendance'), 'Consecutive Absences', 0, 'Reports', 'Returns a list of students who have had a specified number of consecutive days absent.', 'report_consecutiveAbsences.php', 'report_consecutiveAbsences.php', 'Y', 'Y', 'N', 'N', 'N', 'Y', 'N', 'N', 'Y');end
 INSERT INTO `gibbonPermission` (`gibbonRoleID` ,`gibbonActionID`) VALUES ('001', (SELECT gibbonActionID FROM gibbonAction JOIN gibbonModule ON (gibbonAction.gibbonModuleID=gibbonModule.gibbonModuleID) WHERE gibbonModule.name='Attendance' AND gibbonAction.name='Consecutive Absences'));end
 ALTER TABLE `gibbonPerson` CHANGE `studentID` `studentID` VARCHAR(15) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL;end
+UPDATE `gibbonAction` SET name='Manage Roles_all', URLList='role_manage.php,role_manage_add.php,role_manage_edit.php,role_manage_delete.php,role_manage_duplicate.php,role_manage_view.php' WHERE name='Manage Roles' AND gibbonModuleID=(SELECT gibbonModuleID FROM gibbonModule WHERE name='User Admin');end
```

Comment:
Does the precdence of this action need to be incremented to 1, to show it is higher than the view only counterpart?

Reply:
Well caught, yep it should be! Just pushed the change.

Reply:
Wait, changed the wrong precedence! One sec 😬